### PR TITLE
Updates to KFP End-to-end tutorial (Issue #860)

### PR DESCRIPTION
**Pull request by OfficePop**
In response to [kubeflow/website issue 860](https://github.com/kubeflow/website/issues/860), I have made the following edits to the KFP end-to-end GCP tutorial:

1. Updated with recommended kubeflow version from 0.4.1 to 0.5.1 with an html tag (html page has also been added in this PR). In the future, this can be handled by updating the embedded html page instead of directly updating the tutorial.

2. Revised the overview to recommend the user start with Cloud Shell instead of installing Cloud SDK. This reduces the steps the user needs to take and eliminates a potential chokepoint from errors installing or configuring a local copy of the SDK.

3. Removed the instructions to install kubectl locally, as it is already present in Cloud Shell. References to the local copy of kubectl were revised as well. After discussion with  [joeliedtke](https://github.com/joeliedtke) it was agreed to retain the “handy environment variables” procedure when it was identified that it had dependencies later in the tutorially (mainly the user selecting their region/zone).

4. Instructions for granting user access were added to the “Set up your GCP account and SDK” procedure, along with the required roles and a reference link to the full procedure.

5. The commands `apt-get install -y` and `wget bzip2` were removed from the “Set up Python” procedure.

6. A note was added at the end of the “Run the Pipeline” procedure to help the user locate the data for the trained model.

The only item that was not addressed from the original issue was item 6. This requires more familiarity with all aspects of the tutorial than I can provide at this time, and I recommend it be further fleshed out in another issue.

Also, I would suggest updating the screenshot in the “Deploy Kubeflow” procedure. The version number reads “0.4.1”, which out of date. If possible it would be preferable to update it to something generic like <LATEST VERSION> and reduce future maintenance.

I have also appended updates related to [kubeflow/website issue 845](https://github.com/kubeflow/website/issues/845), merging the content from requirements.md into getting-started.md, updating the _redirects page, and deleting requirements.md.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/890)

### CODE REVIEW

File: content/docs/pipelines/tutorials/pipelines-tutorial.md [hunk 1]
```diff
@@ -72,28 +71,20 @@ Follow these steps to set up your GCP environment:
 1. Select or create a project on the [GCP Console][gcp-console].
 1. Make sure that billing is enabled for your project. See the guide to
   [modifying a project's billing settings][billing-guide].
-1. Install the [Cloud SDK][cloud-sdk].
+1. Grant users access to kubeflow with the `Owner` and `IAP-secured Web App User` roles:
```
Comment (joeliedtke) [inline content/docs/pipelines/tutorials/pipelines-tutorial.md]:
```suggestion
1. Grant your team access to Kubeflow by granting them the **Owner** and **IAP-secured Web App User** roles:
```

File: content/docs/pipelines/tutorials/pipelines-tutorial.md [hunk 2]
```diff
@@ -72,28 +71,20 @@ Follow these steps to set up your GCP environment:
...
+
+      ```
+      gcloud projects add-iam-policy-binding <PROJECT>
```
Comment (joeliedtke) [inline content/docs/pipelines/tutorials/pipelines-tutorial.md]: Please remove this script block. I was wrong, project owner cannot be granted from the command line. They need to use Cloud Console.

File: content/docs/pipelines/tutorials/pipelines-tutorial.md [hunk 3]
```diff
@@ -72,28 +71,20 @@ Follow these steps to set up your GCP environment:
...
+      --member user:<EMAIL> --role <ROLE>
+      ```
+    For more details, see [Granting, changing, and revoking access to resources][granting-changing-revoking-access].
```
Comment (joeliedtke) [inline content/docs/pipelines/tutorials/pipelines-tutorial.md]: Suggestion: You can probably streamline this information by focusing on granting access. How about something like "For more details, see the guide to granting access to resources on GCP."

File: content/docs/pipelines/tutorials/pipelines-tutorial.md [hunk 4]
```diff
@@ -158,7 +149,7 @@ Deploy Kubeflow on GCP:
 1. (Optional) If you want to examine your cluster while waiting for the UI to
    be available, you can use `kubectl` to connect to your cluster:
 
-  * Connect your local `kubectl` session to the cluster:
+  * Connect your cloud shell session to the cluster:
```
Comment (joeliedtke) [inline content/docs/pipelines/tutorials/pipelines-tutorial.md]:
```suggestion
  * Connect your Cloud Shell session to the cluster:
```

File: content/docs/pipelines/tutorials/pipelines-tutorial.md [hunk 5]
```diff
@@ -264,7 +255,7 @@ If you don't have a Python 3 environment set up, install
   environment, run the following commands:   
 
     ```bash
-    apt-get update; apt-get install -y wget bzip2
+    apt-get update;
```
Comment (joeliedtke) [inline content/docs/pipelines/tutorials/pipelines-tutorial.md]:
```suggestion
    apt-get update 
```

File: content/docs/pipelines/tutorials/pipelines-tutorial.md (hunk 2 above)
Comment (OfficePop) [reply content/docs/pipelines/tutorials/pipelines-tutorial.md]: I adjusted this, removing the 'Owner' part but leaving the IAP role. Or do they all need to be removed?

**Comment by OfficePop** [comment]
Believe I fixed my mismatched email rules between this account and my CLA on file, adding this comment so the bot runs again.

**Comment by OfficePop** [comment]
> Believe I fixed my mismatched email rules between this account and my CLA on file, adding this comment so the bot runs again.

Turns out I had to update the CLA too. That has been done as well.

**Comment by OfficePop** [comment]
Had some merge conflicts and CLA issues. Closing this PR and issuing a new one to go more smoothly.

**Comment by OfficePop** [comment]
New PR: https://github.com/kubeflow/website/pull/915

**Pull request closed without merging**